File: templates/matlab/protection_fault_scenario_facets.m
function fig = protection_fault_scenario_facets()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('small_multiples', 4009, 'protection and fault analysis: scenario small multiples', 'protection and fault analysis', 'scenario small multiples');
end

File: templates/python/protection_fault_scenario_facets.py
"""protection_fault_scenario_facets: 保护与故障分析场景分面（small-multiples 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='protection and fault analysis: scenario small multiples'):
    return make_template_figure('small_multiples', seed=4009, title=title, domain='protection and fault analysis', topic='scenario small multiples')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
